### ISSUE TITLE
Need a filter icon instead of the word "(filtered)"

### ISSUE THREAD

**Issue by TonyGermaneri** [TonyGermaneri/canvas-datagrid#81]
just what the title says.

**Comment by simnovus** [comment]
Any update on this?

**Comment by TonyGermaneri** [comment]
No, but contributions are always welcome!  I can help point you in the right direction.  I've paused work on this project to start a private project, I will return here soon.  I will still field PRs and make releases.

**Comment by vermadev95** [comment]
> No, but contributions are always welcome! I can help point you in the right direction. I've paused work on this project to start a private project, I will return here soon. I will still field PRs and make releases.

Any help regarding this will be appreciated.

**Comment by TonyGermaneri** [comment]
It would be very much like this
https://github.com/TonyGermaneri/canvas-datagrid/blob/master/lib/draw.js#L799

This is the order arrow drawing function.
https://github.com/TonyGermaneri/canvas-datagrid/blob/master/lib/draw.js#L95

You can also replace the word "filtered" with an unicode glyph.

I don't know why it's not in the docs (another bug), but `args.filterTextPrefix` will change the value of the text displayed.  Try putting new glyphs in it maybe?

I like to find glyphs here:
https://graphemica.com/

Let me know if I can be of any further help.

Cheers,
Tony

**Comment by vermadev95** [comment]
Thank you very much for the help.
Putting `filterTextPrefix:` `\u0264` in the grid property fixed my problem.
